src/app/components/Item/ItemFilter.jsx

```diff
@@ -5,16 +5,17 @@ import FocusTrap from 'focus-trap-react';
 
 import { isOptionSelected, trackDiscovery } from '../../utils/utils';
 
-// export for testing purposes
-export const parseDistinctOptions = options => Array.from(
-  new Set(options.reduce((optionIds, option) => {
-    if (option.id && option.id.length) optionIds.push(option.id);
-    return optionIds;
-  }, [])))
-  .map(id => ({
-    id,
-    label: options.find(option => (option.id === id && option.label.length)).label,
-  }));
+export const parseDistinctOptions = options =>
+  Array.from(
+    new Set(options.reduce((optionIds, option) => {
+      if (Array.isArray(option.label)) return optionIds.concat(option.label);
+      return optionIds.concat([option.label]);
```

Comment:
Should the param now be called `optionLabels` instead of `optionIds`?

Reply:
Changed. In response to the comment above: while working on this I was seeing issues where items were not selected properly because of slightly mismatched labels (e.g. `Available` vs `Available `). It is true this version will potentially have duplicated locations where there are two different labels for the same location, but I think that is better than not seeing them. We should definitely try to improve this by combining locations that are the same, but I think that will require some finesse and this is the simplest working implementation. So we could nix id but I suspect we will want it in the future (although the logic for determining id will change).